tests/test_basis.py

```diff
@@ -142,76 +142,72 @@ def test_chebyshev(self):
             chebyshev(r[:, np.newaxis], l, dr=1)
 
     @pytest.mark.unit
-    def test_zernike_radial(self):
+    def test_zernike_radial(self):  # noqa: C901
         """Test zernike_radial function, comparing to analytic formulas."""
-
-        def test_zernike_radial(self):
```

Comment:
so i found the coverage problem..

Reply:
Yes I must have copied the function definition in a later commit after I tested. Then black formatting finished the job of tricking me